docs/tutorials/radio.rst

```diff
@@ -1,163 +1,149 @@
-for a new incoming message. If the message queue fills up, then new incoming
-messages are ignored.
+	new_message = radio.receive()
+	
+Odebrane wiadomości są przechowywane w kolejce wiadomości. Funkcja ``receive`` 
+wyświetla najstarsze wiadomości z kolejki jako ciąg znaków, robiąc miejsce 
+na nowe wiadomości przychodzące. Jeżeli kolejka się zapełni, wtedy kolejne 
+wiadomości przychodzące będą ignorowane. 
 
-That's really all there is to it! (Although the radio module is also powerful
-enough that you can send any arbitrary type of data, not just strings. See the
-API documentation for how this works.)
+To tak naprawdę wszystko! (Chociaż moduł radia jest wystarczająco mocny aby przesyłać dowolny typa danych, nie tylko ciągi znaków. Sprawdź w dokumentacji API jak to działa.)
 
-Armed with this knowledge, it's simple to make micro:bit fireflies like this:
+	
+Wyposażony w tę wiedzę, łatwo jest stworzyć micro:bit'owe świetliki:
 
 .. include:: ../../examples/radio.py
     :code: python
 
-The import stuff happens in the event loop. First, it checks if button A was
-pressed and, if it was, uses the radio to send the message "flash". Then it
-reads any messages from the message queue with ``radio.receive()``. If there is
-a message it sleeps a short, random period of time (to make the display more
-interesting) and uses ``display.show()`` to animate a firefly flash. Finally,
-to make things a bit exciting, it chooses a random number so that it has a 1 in
-10 chance of re-broadcasting the "flash" message to anyone else (this is how
-it's possible to sustain the firefly display among several devices). If it
-decides to re-broadcast then it waits for half a second (so the display from
-the initial flash message has chance to die down) before sending
-the "flash" signal again. Because this code is enclosed within a ``while True``
-block, it loops back to the beginning of the event loop and repeats this
-process forever.
-
-The end result (using a group of micro:bits) should look something like this:
+
+Następuje import w pętli. Najpierw, sprawdzi czy przycisk A został wciśnięty,
+jeśli tak, to wyśle wiadomość o treści "flash" ("błyskaj"). Następnie wyświetli 
+dowolną wiadomość z kolejki przy pomocy ``radio.receive()``. Jeżeli w kolejce
+jest wiadomość, następuje uśpienie na krótki czas (aby uczynić pokaz bardziej 
+interesującym) i przy pomocy animuje błyski świetlików. Na koniec, aby było 
+jeszcze bardziej ekscytująco, wybiera losowy numer tak, że ma jedną na dziesięć 
+szans na przesłanie wiadomości "flash" dalej do kogokolwiek (dzięki temu jest 
+możliwe utrzymanie pokazu świetlików wśród kilku urządzeń). Jeżeli urządzenie 
+zdecyduje się przesłać wiadomość dalej, czeka pół sekundy (by pierwotnie 
+wyświetlana wiadomości wygasła) przed ponownym wysłaniem wiadomości "flash". 
+Ponieważ ten kod znajduje się w bloku ``while True``, rozpoczyna pętlę 
+od początku i powtarza ją w nieskończoność.
```

Comment:
To jest jakiś bełkot. Bardzo proszę przeczytać swoje tłumaczenia choć raz przed wysłaniem.